### ISSUE TITLE
CMake Tools doesn't reconfigure when editing CMakeLists.txt in a subdirectory

### ISSUE THREAD

**Issue by jasper-vb** [microsoft/vscode-cmake-tools#1579]
### Brief Issue Summary

I have a project structure like this:

```
/
|- CMakeLists.txt
|- src
    |- ...
|- test
    |- CMakeLists.txt
    |- src
        |- ...
```

In the toplevel `CMakeLists.txt`, I have a line `add_subdirectory(test)`.

`cmake.configureOnEdit` is set to true. When I edit the toplevel `CMakeLists.txt`, the project gets reconfigured. When I edit `test/CMakeLists.txt`, it doesn't

### Expected:

1. Edit `test/CMakeLists.txt`
2. Project gets reconfigured

### Apparent Behavior:

1. Edit `test/CMakeLists.txt`
2. Nothing happens

### Platform and Versions

- **Operating System**: Ubuntu 20.04 via WSL2 on Windows 10
- **CMake Version**: 3.16.3
- **VSCode Version**: 1.51.1
- **CMake Tools Extension Version**: 1.5.3
- **Compiler/Toolchain**: clang 11

### Other Notes/Information

This is semi-related to #1374, but I don't want to create multiple build directories -- I just want my one build directory to be reconfigured when I edit either one of the `CMakeLists.txt` files.

Ideally, CMake Tools would auto-detect which subdirectory `CMakeLists.txt` files are used, but I would also be fine with an option to specify which `CMakeLists.txt` files CMake Tools should watch for changes.

**Comment by bobbrow** [comment]
Thank you for reporting this issue.  I think we have what we need, but if we need more information we will let you know.

**Comment by bobbrow** [comment]
The fix for this issue is available in 1.10 "pre-release".  You can help us validate it by:
1. Opening the extensions panel in VS Code
2. Selecting the CMake Tools extension
3. Clicking the "Switch to Pre-release Version" button

![image](https://user-images.githubusercontent.com/12818240/156218538-7bce4a1f-4fc9-43cc-afb0-7f423aa5980e.png)

**Comment by jasper-vb** [comment]
It seems to work even without switching to the pre-release version.


- **Operating System:** Ubuntu 20.04 via WSL2 on Windows 10
- **CMake Version:** 3.16.3
- **VSCode Version:** 1.64.2
- **CMake Tools Extension Version:** 1.9.2
- **Compiler/Toolchain:** clang 13
